tests/components/automation/test_init.py

```diff
@@ -359,41 +359,54 @@ def test_services(self):
 
         automation.toggle(self.hass, entity_id)
         self.hass.block_till_done()
+        self.hass.bus.fire('test_event')
```

Comment:
If we need this, we're hiding the fact that something else is wrong. I'm reverting this change.